example/server.js

```diff
@@ -0,0 +1,14 @@
+var sync = require('../lib');
+
+var mongodbConnectionString = 'mongodb://127.0.0.1:27017/sync';
+var redisUrl = 'redis://127.0.0.1:6379';
+
+sync.api.connect(mongodbConnectionString, {}, redisUrl, function(){});
+
+sync.api.getEventEmitter().on('sync:ready', function() {
+  console.log('sync ready');
+
+  sync.api.init('myDataset', {
+    syncFrequency: 10 // seconds
+  }, function() {});
```

Comment:
I like idea of single simplified server file instead of full app in #1 
Going to extend it a little bit after merge.